docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+Table: Key Usage for ECC Public Keys
+
+| __Key Usage__      | __Permitted__ | __Required__     |
+| ----               | -             | -                |
+| `digitalSignature` | Y             | MUST             |
+| `nonRepudiation`   | N             | --               |
+| `keyEncipherment`  | N             | --               |
+| `dataEncipherment` | N             | --               |
+| `keyAgreement`     | N             | --               |
+| `keyCertSign`      | N             | --               |
+| `cRLSign`          | N             | --               |
+| `encipherOnly`     | N             | --               |
+| `decipherOnly`     | N             | --               |
+
+##### 7.1.2.6.12 Subject Alternative Name
+
+For Subscriber Certificates, the Subject Alternative Name MUST be present and MUST contain at least one `dNSName` or `iPAddress` `GeneralName`. See below for further requirements about the permitted fields and their validation requirements.
+
+Table: `GeneralName` within a `subjectAltName` extension
+
+| __Name Type__               | __Permitted__ | __Validation__ |
+| ---                         | -             | ------         |
+| `otherName`                 | N             | -              |
+| `rfc822Name`                | N             | -              |
+| `dNSName`                   | Y             | MUST contain the Fully-Qualified Domain Name. The Issuing CA MUST confirm that the Applicant controls or has been granted the right to use the Domain Name through a method specified in [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). As an exception to the requirement of RFC 5280 requirements, Wildcard FQDNs are permitted. All other domain labels MUST be in the "preferred name syntax" and thus MUST NOT contain underscore characters ("\_"). MUST NOT contain an Internal Name. |
+| `x400Address`               | N             | -              |
+| `directoryName`             | N             | -              |
+| `ediPartyName`              | N             | -              |
+| `uniformResourceIdentifier` | N             | -              |
+| `iPAddress`                 | Y             | MUST contain the IPv4 or IPv6 address that the CA has confirmed the Applicant controls or has been granted the right to use through a method specified in [Section 3.2.2.5](#3225-authentication-for-an-ip-address). MUST NOT contain a Reserved IP Address. |
+| `registeredID`              | N             | -              |
+
+**Note**: If the `subject` field of the certificate is an empty SEQUENCE, this extension MUST be marked critical, as specified in [RFC 5280, Section 4.2.1.6](https://tools.ietf.org/html/rfc5280#section-4.2.1.6).
+
+#### 7.1.2.7 OCSP Responder Certificate Profile
+
+If the Issuing CA does not directly sign OCSP responses, it MAY make use of an OCSP Authorized Responder. The Issuing CA of the Responder MUST be the same as the Issuing CA for the Certificates it provides responses for.
```

Comment:
I'm not sure I agree that capitalization means it belongs in definitions, considering this term is established by RFC 6960. It seems like you're suggesting a definition to just point to the OCSP specification, is that correct? Our definitions don't define "CA" or "CA Certificate", so I'm not sure we have to strictly do that.

My goal was not to duplicate definitions (which, as we saw with cross-certificate, caused confusion). Would it be clearer by having a normative reference on RFC 5280 for terms as well?

Reply:
It would be nice to try to address this confusion. The SCWG failed to address and clarify the "CA" term in the past years, but we currently identify "CA" in section 1.6.2 as "Certification Authority" which is a defined term in section 1.6.1. The BRs also have "Subordinate CA" and "Root CA" defined. 

My point is that if we use a term in capitals, the expectation by readers is that these terms are defined in 1.6.1. Alternatively, perhaps it might be acceptable to add a clear reference in 1.6.1 pointing to definitions in other normative documents like RFC 5280 and 6960 but then we must also state what happens in case of conflicts of these definitions (which document takes priority). Alternatively, we could remove the capitalization from "Authorized Responder" as it currently exists in 6960.

Reply:
My objection to adding it to the definitions section is I think the same reason you're supportive of it, which makes it tricky :) We already know that the definitions of "Subordinate CA" and "Root CA" have caused confusion (re: Cross-Certificate), and so I'm very keen not to repeat definitions, and I'm not sure "definition: see Y" is all that useful.

I've hopefully found a reasonable balance here by tweaking 7.1.2.7, but let me know if that doesn't fully address your concerns.